scripts/ci/run_perftest.sh

```diff
@@ -2,32 +2,41 @@
 
 echo "Running as user: $(whoami)"
 
+# Source env variables
+. /home/ubuntu/vars.sh
+
+# Install influxdb
+DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes /home/ubuntu/influxdb2*amd64.deb
+systemctl start influxdb
+DATASET_DIR=/mnt/ramdisk
+mkdir -p "$DATASET_DIR"
+mount -t tmpfs -o size=32G tmpfs "$DATASET_DIR"
+
+# set up influxdb
+export INFLUXDB2=true
+export TEST_ORG=example_org
+export TEST_TOKEN=token
+result="$(curl -s -o /dev/null -H "Content-Type: application/json" -XPOST -d '{"username": "default", "password": "thisisnotused", "retentionPeriodSeconds": 0, "org": "'"$TEST_ORG"'", "bucket": "unused_bucket", "token": "'"$TEST_TOKEN"'"}' http://localhost:8086/api/v2/setup -w %{http_code})"
+if [[ "$result" != "201" ]] ; then
```

Comment:
I guess this must work based on the dry run, but I know that sometimes there's a period of time between when the `influxd` process is started successfully and when it can actually respond to http requests. Does `systemctl start influxdb` not return until `influxd` is actually able to respond to http requests?

Reply:
I believe the start script tries to wait for influxdb to actually start, yes.

Reply:
Oh, yep, it definitely does: https://github.com/influxdata/influxdb/blob/fa9ba8e86f4f21344cd1edd66fc8bed07331eb10/scripts/influxd-systemd-start.sh#L23-L28